go/vumitools/app_worker.py

```diff
@@ -211,53 +212,25 @@ def _find_outboundmessage_for_event(self, event):
 
     _EVENT_OUTBOUND_CACHE_KEY = "outbound_message_json"
 
-    def _get_outbound_from_event_cache(
-            self, event, key=_EVENT_OUTBOUND_CACHE_KEY):
-        """ Retrieve outbound message from the cache on an event.
-
-        :type event:
-            TransportEvent
-        :param event:
-            Event to look up the outbound message on.
```

Comment:
Also, these changes seem a bit unrelated -- just checking that that assumption is correct?